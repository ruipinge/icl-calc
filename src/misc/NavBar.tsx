export const NavBar = ({ resetForm }: { resetForm: (a?: any) => void }) => (
  <nav className="navbar navbar-expand navbar-dark bg-dark fixed-top">
    <div className="container">
      <a className="navbar-brand" href={process.env.PUBLIC_URL || '/'}>
        ICL Size Calc
      </a>
      <form className="form-inline">
        <button
          type="button"
          className="btn btn-danger"
          onClick={() => {
            resetForm();
          }}
        >
          Reset
        </button>
      </form>
    </div>
  </nav>
);
