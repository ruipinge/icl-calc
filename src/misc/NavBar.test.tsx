import { NavBar } from './NavBar';
import { render } from '@testing-library/react';

it('renders without crashing', () => {
  const { asFragment } = render(<NavBar resetForm={() => {}} />);
  expect(asFragment()).toMatchSnapshot();
});
