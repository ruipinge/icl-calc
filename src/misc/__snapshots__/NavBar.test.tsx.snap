// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<DocumentFragment>
  <nav
    class="navbar navbar-expand navbar-dark bg-dark fixed-top"
  >
    <div
      class="container"
    >
      <a
        class="navbar-brand"
        href="/"
      >
        ICL Size Calc
      </a>
      <form
        class="form-inline"
      >
        <button
          class="btn btn-danger"
          type="button"
        >
          Reset
        </button>
      </form>
    </div>
  </nav>
</DocumentFragment>
`;
